controllers/imagejob/imagejob_controller.go

```diff
@@ -325,6 +325,13 @@ func (r *Reconciler) handleNewJob(ctx context.Context, imageJob *eraserv1.ImageJ
 		return err
 	}
 
+	var nodeNames []string
```

Comment:
I would try checking memory usage without this change (unclear to me if it is actually helping).

Reply:
yes thats correct! I removed this